motioneye/handlers.py

```diff
@@ -1669,7 +1669,7 @@ def on_finish(self):
                 if os.path.isfile(f) and os.stat(f).st_atime <= stale_time:
                     os.remove(f)
         except:
-            logging.error('could not delete temp file')
+            logging.exception('could not delete temp file')
```

Comment:
One small remark: could we use `logging.error(..., exc_info=True)` for the sake of consistency with the rest of the code?

Reply:
Done.